state/endpoint_bindings.go

```diff
@@ -264,7 +267,7 @@ func (b *Bindings) updateOps(txnRevno int64, newMap map[string]string, newMeta *
 // validateForMachines checks whether the required space
 // assignments are actually feasible given the network configuration settings
 // of the machines where application units are already running.
-func (b *Bindings) validateForMachines() error {
+func (b *Bindings) validateForMachines(oldDefaultSpaceID string) error {
```

Comment:
Suggest new arg name, it is too similar to network.DefaultSpaceID. oldSpaceIDOfDefaultEndpoint?